### ISSUE TITLE
Add anchor_link module to GovCMS

### ISSUE THREAD

**Issue by adresden-opc** [govCMS/GovCMS#411]
** What value does this module/package add to GovCMS?

The ability to add name anchors and link to them is basic functionality.

** Please provide a brief outline of what this module does.

Adds a button to the ckeditor toolbar. Clicking the button adds a named anchor element to the edited page.

** Who does this module benefit:
[ ] end users
[ *] content editors
[ ] site builders
[ ] themers
[ ] developers
 
** How could you provide/replicate the functionality of this module using alternative methods, eg in your theme?

None

** If this module styles or alters HTML or JavaScript output, can the functionality be provided via the theme? What alternatives have you considered.

There are none.

** What is the maintenance and support status of the module. Describe the issue queue activity.

[8.x-2.5](https://www.drupal.org/project/anchor_link/releases/8.x-2.5) - 9 February 2021

** What permissions are needed to utilise the module (and are any new permissions provided by the module)?

None

** Does the module modify the database structure and/or store additional metadata on nodes or other entities? If so, why? What are the risks for future updates?

None

** Is the module designed to capture anonymous user data?

no.

** Is the output of the module typically fully cacheable? Would the inclusion of this module potentially render pages uncacheable.

n/a

** What is your assessment of the quality of this module, the contribution history of the module's maintainers, and the uptake of the module within the Drupal community?

Stable. Used correctly, it is a solid module.

** Additional information

**Comment by thisisalistairsaccount** [comment]
Hi @adresden-opc we had this raised previously when we were working on Drupal 8 https://github.com/govCMS/GovCMS8/issues/393 

We are currently looking at this internally. To note, there's a dependency on the CKEditor Fake Objects https://www.drupal.org/project/fakeobjects that we also have to look as part of this.

**Comment by pellegs** [comment]
Issue raised with fakeobjects a blocker at this time
https://www.drupal.org/project/fakeobjects/issues/3118327

**Comment by thisisalistairsaccount** [comment]
For visibility, the issue raised at https://www.drupal.org/project/fakeobjects/issues/3118327 is still RTBC and hasn't been included.

To note, with the move away from CKEditor 4 to 5 (part of the Drupal 9.5 branch) also means the current release of acnhor_link (as of this comment) is only CKEditor 4 compatible. There is an issue raised at https://www.drupal.org/project/anchor_link/issues/3206676 around this, but is still early days. 

Issue is still blocked.

**Comment by tumendelger** [comment]
Hello @thisisalistairsaccount, 

Any update on this one? Will there be an inclusion of this module?

**Comment by thisisalistairsaccount** [comment]
Hi @tumendelger the same issues I've last flagged are still outstanding.

**Comment by StephenOllman** [comment]
Whilst a useful tool to have in the editor toolbar, anchor links are pretty easy to achieve using plain HTML.

**Comment by brett-sargeant** [comment]
It looks like it's been made compatible with CK5

**Comment by dibs** [comment]
This would be a good addition for editors who may not feel as confident with HTML.

**Comment by StephenOllman** [comment]
Agreed, this would benefit authors that have limited HTML experience.

**Comment by kobus-vanwyk** [comment]
Alternative solution: https://www.drupal.org/project/ckeditor5_plugin_pack  see Bookmarks plugin for more details.
See ticket https://github.com/govCMS/GovCMS/issues/840 for more information.

The question to the community is, do you think this is valuable to be included?

**Comment by kobus-vanwyk** [comment]
We have a beta release at [anchor_link  3.0.0-beta1]( https://www.drupal.org/project/anchor_link/releases/3.0.0-beta1) but no stable covering CKE5

**Comment by yvonne-norris** [comment]
We've recommenced our investigation, there is now a stable release of the module

**Comment by jackwrfuller** [comment]
Anchor Link has been added to the GovCMS distribution, and will be included in the upcoming Drupal 11 version.
